layers/libVkLayer_khronos_shader_object.map

```diff
@@ -2,10 +2,6 @@
   global:
     vkGetInstanceProcAddr;
     vkGetDeviceProcAddr;
-    vkEnumerateInstanceExtensionProperties;
-    vkEnumerateInstanceLayerProperties;
-    vkEnumerateDeviceExtensionProperties;
-    vkEnumerateDeviceLayerProperties;
```

Comment:
NDK 26 will fail to build due to stricter error checking:

https://github.com/android/ndk/releases/tag/r26-rc1

>  Version scripts that name public symbols that are not present in the library will now emit an error by default for ndk-build and the CMake toolchain file.

```
FAILED: layers/libVkLayer_khronos_shader_object.so 
: && /home/juan/Android/Sdk/ndk/26.01/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++ --target=aarch64-none-linux-android26 --sysroot=/home/juan/Android/Sdk/ndk/26.01/toolchains/llvm/prebuilt/linux-x86_64/sysroot -fPIC -DANDROID -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security -fexceptions -frtti -stdlib=libc++ -g  -fno-limit-debug-info  -Wl,--build-id=sha1 -Wl,--no-rosegment -Wl,--no-undefined-version -Wl,--fatal-warnings -Wl,--gc-sections -Qunused-arguments -Wl,--no-undefined -fuse-ld=lld   -Xlinker --version-script=/home/juan/projects/vel/layers/libVkLayer_khronos_shader_object.map -shared  -o layers/libVkLayer_khronos_shader_object.so layers/CMakeFiles/VkLayer_khronos_shader_object.dir/shader_object.cpp.o  utils/libVkExtLayer_utils.a  -llog   -static-libstdc++ -latomic -lm && :
ld.lld: error: version script assignment of 'global' to symbol 'vkEnumerateInstanceExtensionProperties' failed: symbol not defined
ld.lld: error: version script assignment of 'global' to symbol 'vkEnumerateInstanceLayerProperties' failed: symbol not defined
ld.lld: error: version script assignment of 'global' to symbol 'vkEnumerateDeviceExtensionProperties' failed: symbol not defined
ld.lld: error: version script assignment of 'global' to symbol 'vkEnumerateDeviceLayerProperties' failed: symbol not defined
```

Reply:
IE exporting these symbols isn't valid because they aren't defined in the library and can result in bad behavior.

Reply:
FYI @ziga-lunarg